xde-tests/src/lib.rs

```diff
@@ -0,0 +1,464 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+// Copyright 2024 Oxide Computer Company
+
```

Comment:
This file is mostly `xde-tests/tests/loopback.rs`, moved into a library usable from kbench.